### ISSUE TITLE
Audit tables column `diffs` type unexpectedly changed from JSON to TEXT

### ISSUE THREAD

**Issue by pniep** [DamienHarper/auditor#52]
Hi @DamienHarper,

After update doctrine wants to change type of all `diffs` columns in audit tables from `json` to `text`. [db: postgres, json support: yes]
I'm not sure that was intended.

Fragment below forces TEXT as the type of all JSON columns when database actually supports json columns .
https://github.com/DamienHarper/auditor/blob/adc9fa1d0059dc1074cf9c6efc3ca84fcf9999bb/src/Provider/Doctrine/Persistence/Schema/SchemaManager.php#L192
```
if (DoctrineHelper::getDoctrineType('JSON') === $struct['type'] && $isJsonSupported) {
    $type = DoctrineHelper::getDoctrineType('TEXT');
} else {
    $type = $struct['type'];
}
```

It looks like a bug. As commit message for these lines says the intent was to add compatibility for older versions of mariadb that don't support json.

**Comment by dappa** [comment]
Any chance to revisit this issue?
It seems to be present on 2.0.5 still.

**Comment by DamienHarper** [comment]
@dappa @pniep Fixed in `master` and `2.4.7`. 
Sorry it took so long.
